dbscan/dbscan.go

```diff
@@ -21,13 +21,13 @@ type Rows interface {
 // ScanAll is a package-level helper function that uses the DefaultAPI object.
 // See API.ScanAll for details.
 func ScanAll(dst interface{}, rows Rows) error {
-	return errors.WithStack(DefaultAPI.ScanAll(dst, rows))
+	return DefaultAPI.ScanAll(dst, rows)
```

Comment:
didn't wrap this because there's no point IMO: context is needed when we need to differentiate multiple branches that lead to errors, like in `parseSliceDestination` below. there is only one branch here.

Reply:
Sounds good